bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -728,6 +728,29 @@ private function addDocumentsNode(ArrayNodeDefinition $rootNode)
 
         $documentsNode
             ->children()
+                 ->arrayNode('doctype')
```

Comment:
```suggestion
                 ->arrayNode('doc_types')
```
Since we use camelCase for the class name, I'd use snake_case for the config node.